tests/test_run_config_measurement.py

```diff
@@ -208,6 +210,48 @@ def test_is_better_than_consistency(self):
         self.assertTrue(self.rcm2.is_better_than(self.rcm3))
         self.assertFalse(self.rcm3.is_better_than(self.rcm2))
 
+    def test_compare_measurements(self):
+        """
+        Test to ensure compare measurement function returns
+        the correct magnitude
+        # RCM4's throughput is 1000
+        # RCM5's throughput is 2000
+        # Therefore, the magnitude is (RCM5 - RCM4) / avg throughput
+        #                             (2000 - 1000) / 1500
+        """
+        rcm4_vs_rcm5 = self.rcm4.compare_measurements(self.rcm5)
+        self.assertEqual(rcm4_vs_rcm5, 1000 / 1500)
+
+        rcm5_vs_rcm4 = self.rcm5.compare_measurements(self.rcm4)
+        self.assertEqual(rcm5_vs_rcm4, -1000 / 1500)
+
+    def test_is_passing_constraints_none(self):
+        """
+        Test to ensure constraints are reported as passing
+        if none were specified
+        """
+        self.assertTrue(self.rcm5.is_passing_constraints())
+
+    def test_is_passing_constraints(self):
+        """
+        Test to ensure constraints are reported as
+        passing/failing if model is above/below
+        throughput threshold
+        """
+        self.rcm5.set_model_config_constraints([{
```

Comment:
general python testing question -- does this affect the value of self.rcm5 for all future tests in this file? Could this fail if the test runner decides to run this before test_is_passing_constraints_none?

Reply:
It should not as they are created in the setup of each test (and theoretically destroyed in the teardown)